.tx/config

```diff
@@ -1,6 +1,14 @@
 [main]
 host = https://www.transifex.com
 
+[o:openmrs:p:openmrs-esm-patient-chart:r:esm-form-engine-app]
```

Comment:
@ibacher, is this `o:` prefix necessary?

Reply:
Yes. It's [Transifex's syntax](https://developers.transifex.com/docs/migrating-from-older-versions-of-the-client#add). (`o:` for org, `p:` for project, `r:` for resource)

Reply:
After checking the results and command instructions, it seems that the new version of Transifex requires an additional parameter to be configured: "resource_name". Otherwise, the new added resource name displayed on Transifex will default to "en.json". 

I will submit another PR soon.

![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/31535803/05338768-bcf6-4e61-b265-dc7cc54d5ca2)

Reply:
That's probably because `en.json` is treated as the "master list" of strings to be translated.

Reply:
I don't quite understand. How should we go about solving this issue?

Reply:
@befantasy Yeah, I don't think it's determined by the configuration file or it may be that the resources need to be manually renamed after they were created?

Reply:
@ibacher Oh, I see. Thank you for your patient explanation. So do you know how and by whom should the manual renaming be conducted?  I'm stuck with this issue.